bubble-wrap-style.yaml

```diff
@@ -3876,6 +3085,29 @@ layers:
                         style: italic
                         size: 10px
                         stroke: { color: global.text_stroke_address, width: 4 }
+        entrance-labels:
+            filter:
+                $zoom: { min: 20 }
+                any:
+                    - kind: entrance
+                all:
+                    - function() { return global.text_visible_address; }
+            draw:
+                mapzen_icon_library:
+                    size: [[13, [95%,95%]], [16, [100%,100%]], [18, [105%,105%]]]
+                    sprite: generic #function() { return feature.kind; }
+                    sprite_default: generic
```

Comment:
If sprite is always `generic` (line above), then `sprite_default` is extraneous here.